vscode-extension/static/env_template.env

```diff
@@ -1,15 +1,10 @@
-# Uncomment the following lines to set these common env variables for 
-# accessing generative AI models. If your model requires other env variables
-# you can also set those in this file
+# This is your .env file to specify keys from model providers. Obtain your keys:
```

Comment:
We should say something more like "common keys include" because they don't necessarily need these, it depends on the model they're using